mandatory/3-playing-computer.js

```diff
@@ -7,12 +7,19 @@
   Answer the following questions:
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
+      b is not declared
   2. Remove the line that throws the error.
+      commented out console.log(b);
   3. What is printed to the console?
+    2, 6, 4, 9, 6, 13, 8
   4. How many times is "f1" called?
+      2
   5. How many times is "f2" called?
+      3
   6. What value does the "a" parameter take in the first "f1" call?
+      7
```

Comment:
This is actually 1. The 'a' parameter is passed in as the 'i' variable in the f1 on the second iteration of the for loop